eclipse-collections-api/src/main/java/org/eclipse/collections/api/multimap/MutableMultimap.java

```diff
@@ -69,6 +69,17 @@ default boolean putAllPairs(Iterable<? extends Pair<? extends K, ? extends V>> p
         return changed;
     }
 
+    default Iterable<? extends V> getIfAbsentPut(K key, Iterable<? extends V> value)
+    {
+        boolean hasKey = this.containsKey(key);
+        if (hasKey)
+        {
+            this.putAll(key, value);
+            return value;
+        }
+        return this.get(key);
```

Comment:
Commenting here since I just saw that the other copy of this code review is closed. This looks like it's only putting if the key is present, which looks backwards.

Reply:
Yep. You're right. Will fix.